### DISCUSSION TITLE
New task: Question for a transportation network in which a station is located

### DISCUSSION THREAD

**Discussion by gisfun92** [streetcomplete/StreetComplete#5856] [Ideas]
I would like to add a new task to the stations. The transport association should be entered. 
Example: Which transport association can this stop be assigned to?

**Comment by matkoniecz** [comment]
quest suggestion template ( https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&projects=&template=quest-suggestion.md ) from https://github.com/streetcomplete/StreetComplete/issues/new/choose (new issue button) has helpful list of what is needed for a new quest

see also https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md

(I have no ideas how "transport association" would be tagged in OSM, for start)
